src/mongocxx/database.hpp

```diff
@@ -162,9 +162,30 @@ class MONGOCXX_API database {
     bsoncxx::document::value stats();
 
     ///
+    /// Sets the read_concern for this database.
+    ///
+    /// @note Modifications at this level do not affect existing collection instances that have come
+    /// from this database, but do affect new ones. New collections will receive a copy of the
+    /// new read_concern for this database upon instantiation.
+    ///
+    /// @param rc
+    ///   The new @c read_concern
+    ///
+    /// TODO add link to docs once they exist.
+    ///
+    void read_concern(class read_concern rc);
+
+    ///
+    /// The current read concern for this database.
+    ///
+    /// @return the current read_concern
+    ///
+    stdx::optional<class read_concern> read_concern() const;
```

Comment:
Can you also add a comment here that mentions CXX-758, so we remember to change this once that is fixed?